src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -927,10 +927,10 @@ void c_typecheck_baset::typecheck_side_effect_statement_expression(
 
     code_function_callt &fc=to_code_function_call(last);
 
-    const auto &return_type = to_code_type(fc.function().type()).return_type();
+    const auto &r = to_code_type(fc.function().type()).return_type();
```

Comment:
Suggestions: `return_type_ref`, `call_return_type` ?

Reply:
I have just removed the temporary instead.